### PR TITLE
Merge conflict choose language

### PR DESCRIPTION
**Pull request by VonLatvala**
Please do review carefully

### CODE REVIEW

File: src/service.js [hunk 1]
```diff
@@ -57,29 +57,52 @@ const choicesUrl = `${baseUrl}/${organizationName}/observation-questions-choices
 const categoryUrl = `${baseUrl}/${organizationName}/observation-categories/links/${linkName}`;
 const placeUrl = `${baseUrl}/${organizationName}/places/links/${linkName}`;
 const postUrl = `${baseUrl}/${organizationName}/observations/links/${linkName}`
+const availableLangUrl = `${baseUrl}/${organizationName}/available-languages`
+const currentLangUrl = `${baseUrl}/${organizationName}/current-language`
+
+// not used
```
Comment (andersnylund) [inline src/service.js]: Method `getCurrentLang()` is not used

File: src/components/Language.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+// import Button from '@material-ui/core/Button';
```
Comment (andersnylund) [inline src/components/Language.jsx]: Remove old commented code

File: src/App.jsx [hunk 3]
```diff
@@ -32,31 +34,47 @@ class App extends React.Component {
...
     }
 
-    setCatAndLoc = async () => {
+    setInfo = async () => { // better name ideas?
         const cat = await questionService.getCategory()
         const loc = await questionService.getPlace()
+        const lang = await questionService.getLanguages()
+
+        // Sets the default language as the language, and changes it if default not included in API's languages
+        var langId = DEFAULT_LANG_ID
```
Comment (andersnylund) [inline src/App.jsx]: Avoid `var`, use `let` instead

Comment (jallusparta) [reply src/App.jsx]: Changed

File: src/components/Language.jsx (hunk 2 above)
Comment (jallusparta) [reply src/components/Language.jsx]: Removed

File: src/service.js (hunk 1 above)
Comment (jallusparta) [reply src/service.js]: Method and variables it uses deleted.

**Pull request merged**